src/SubNav.tsx

```diff
@@ -46,12 +46,37 @@ const SubNav = styled(SubNavBase)`
   ${sx};
 `
 
-SubNav.Links = props => <Flex {...props} />
+type StyledSubNavProps = {
+  actions?: React.ReactNode
+  align?: 'right'
+  full?: boolean
+  label?: string
+} & ComponentProps<typeof SubNavBase>
 
-SubNav.Link = styled.a.attrs(props => ({
+function SubNav({actions, className, children, label, theme, ...rest}: StyledSubNavProps) {
+  const classes = classnames(className, 'SubNav')
+  return (
+    <SubNavBase className={classes} aria-label={label} {...rest}>
+      <div className="SubNav-body">{children}</div>
+      {actions && <div className="SubNav-actions">{actions}</div>}
+    </SubNavBase>
+  )
+}
+
+const SubNavLinks = (props: SystemFlexProps & SystemCommonProps & SxProp) => <Flex {...props} />
```

Comment:
@colebemis I'm having a bit of difficulty finding the correct type for this that won't generate a type error - any thoughts?

![image](https://user-images.githubusercontent.com/16739070/107409075-5d3e3880-6ad1-11eb-9703-722be7a53334.png)